web/lib/components/flashcards/useAllSets.js

```diff
@@ -21,6 +22,7 @@ export function useAllSets({
 }) {
   const queryClient = useQueryClient()
   const queryKey = ['flashcard-sets', user?.id, page]
+  const [total, setTotal] = useState(0)
 
   const {
     data: flashcardSets,
```

Comment:
Then destruct the result `data: { set: flashcardSets, total }`

Reply:
the issue here is that `data` can be `undefined`. Destruct `data` will resolve in error

Reply:
Try `data: { set: flashcardSets, total } = {}`

Reply:
how about `data: { flashcardSets, total} = { flashcardSets: [], total: 0 }`